content/administration/maintain/azure_oauth.rst

```diff
@@ -188,24 +188,101 @@ configuration automatically loads the :guilabel:`token` in Odoo, and a tag stati
 
+When using this configuration, every email that is sent from the database will use the address of
+the configured `notification` mailbox. However it should be noted that the name of the sender will
+appear but their email address will change:
+
+.. image:: azure_oauth/from-name-remain.png
+   :align: center
+   :alt: Name from real sender with static email.
+
+.. example::
+   Single outgoing mail server configuration:
+
+   - Outgoing mail server **username** (login) = `notifications@example.com`
+   - Outgoing mail server  :guilabel:`FROM Filtering` = `notifications@example.com`
+   - `mail.catchall.domain` in system parameters = `example.com`
+   - `mail.default.from` in system parameters = `notifications`
+
+User-specific (multiple user) configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to a generic email server, individual email servers can be set up for users in a
+database. These email addresses must be set as the :guilabel:`FROM Filtering` on each individual
+server for this configuration to work.
+
+This configuration is the more difficult of the two Microsoft Azure configurations, in that it
+requires all users configured with email servers to have access rights to settings in order to
+establish a connection to the email server.
+
+Setup
+*****
+
+Each user should have a separate email server set up. The :guilabel:`FROM Filtering` should be set
+so that only the user's email is sent from that server. In other words, only a user with an email
+address that matches the set :guilabel:`FROM Filtering` is able to use this server.
+
+.. seealso::
+   For more information on the from filter visit :ref:`this documentation
+   <email_communication/default>`.
+
+A fallback server must be setup to allow for the sending of :guilabel:`notifications`. :ref:`See
+this documentation <azure_oauth/notifications>`. The :guilabel:`FROM Filtering` for this server
+should have the value of the `{mail.default.from}@{mail.catchall.domain}`.
+
+.. note::
+   The :guilabel:`System Parameters` can be accessed by activating
+   :doc:`../../applications/general/developer_mode` in the :menuselection:`Settings --> Technical
+   --> Parameters --> System Parameters` menu.
+
+.. important::
+   The configuration for this transactional email server can work alongside an outgoing mass-mailing
+   email server. The :guilabel:`FROM Filtering` for the mass-mailing email server can remain empty,
+   but it's require to be added in the Settings of the email marketing application.
+
+   .. seealso::
+      For more information on setting the mass-mailing email server visit :ref:`this documentation
+      <email_communication/mass_mails>`.
+
+.. example::
+   Multiple user outgoing mail server configuration:
+
+   - Notifications mailbox
+      - Outgoing mail server #1 **username** (login) = `notifications@example.com`
+      - Outgoing mail server #1 :guilabel:`FROM Filtering` = `notifications@example.com`
```

Comment:
Can you put this Notifications mailbox AFTER the 2 User ?
- Visually speaking it makes more sense (this is a fallback)
- it matches the explanation given previously (first configure 1 OMS / user then a fallback (...) )

Reply:
Added this in, resolved issue.